server/schema/donations.sql

```diff
@@ -4,7 +4,7 @@ DROP TABLE donations;
 CREATE TYPE status AS ENUM ('pending', 'approved', 'denied', 'flagged', 'scheduling', 'scheduled', 'picked up', 'failed');
 
 CREATE TABLE donations (
-   id SERIAL PRIMARY KEY,
+   id PRIMARY KEY,
```

Comment:
because we're changing the id to be a 6 digit alphanumeric string, change this to a `varchar(6)` and update it in _pgadmin_ as well.